### PR TITLE
Implemented missing tests.

### PR DESCRIPTION
**Pull request by marcin-cebo**

### CODE REVIEW

File: pubnub-chat-impl/src/commonMain/kotlin/com/pubnub/chat/internal/channel/BaseChannel.kt [hunk 1]
```diff
@@ -754,6 +754,10 @@ abstract class BaseChannel<C : Channel, M : Message>(
         typingSent = value
     }
 
+    internal fun getTypingSent(): Instant? {
```
Comment (wkal-pubnub) [inline pubnub-chat-impl/src/commonMain/kotlin/com/pubnub/chat/internal/channel/BaseChannel.kt:757]: nit: you can just change `val typingTest` to internal

File: pubnub-chat-impl/src/commonTest/kotlin/com/pubnub/integration/ChatIntegrationTest.kt [hunk 2]
```diff
@@ -574,6 +577,41 @@ class ChatIntegrationTest : BaseChatIntegrationTest() {
...
+    @Test
+    fun setRestrictionThenUnset() = runTest(timeout = 10.seconds) {
+        val userId = someUser.id
+        val channelId = channel01.id
+        val banned = CompletableDeferred<Unit>()
+        val unbanned = CompletableDeferred<Unit>()
+        val restrictionBan = Restriction(userId = userId, channelId = channelId, ban = true, reason = "rude")
+        val restrictionUnban = Restriction(userId = userId, channelId = channelId, ban = false, mute = false, reason = "ok")
+        pubnub.test(backgroundScope, checkAllEvents = false) {
+            var dispose: AutoCloseable? = null
```
Comment (wkal-pubnub) [inline pubnub-chat-impl/src/commonTest/kotlin/com/pubnub/integration/ChatIntegrationTest.kt:589]: this is never used

**Review by wkal-pubnub — APPROVED** [review]

File: pubnub-chat-impl/src/commonMain/kotlin/com/pubnub/chat/internal/channel/BaseChannel.kt (hunk 1 above)
Comment (marcin-cebo) [reply pubnub-chat-impl/src/commonMain/kotlin/com/pubnub/chat/internal/channel/BaseChannel.kt:757]: Makes sense. I will do this and remove getTypingSend and setTypingSend methods.

File: pubnub-chat-impl/src/commonTest/kotlin/com/pubnub/integration/ChatIntegrationTest.kt (hunk 2 above)
Comment (marcin-cebo) [reply pubnub-chat-impl/src/commonTest/kotlin/com/pubnub/integration/ChatIntegrationTest.kt:589]: Good catch. I will remove it.

**Pull request merged**